taxonomies/additives.result.txt

```diff
@@ -27462,22 +27554,22 @@ vegan:en: no
 vegetarian:en: no
 wikidata:en: Q18967189
 
-en:E543, Calcium sodium polyphosphate
+en:E543, E543 food additive
 bg:E543, E543 food additive
 cs:E543, E543 food additive
 da:E543, E543 food additive
 de:E543, E543 food additive
 el:E543, E543 food additive
 es:E543, E543 food additive
 et:E543, E543 food additive
-fi:E543, Kalsiumnatriumpolyfosfaatti, Kalsiumnatriumpolyfosfaattia
-fr:E543, Polyphosphate de sodium de calcium
+fi:E543, E543 food additive
+fr:E543, E543 food additive
 hu:E543, E543 food additive
 it:E543, E543 food additive
 lt:E543, E543 food additive
 lv:E543, E543 food additive
 mt:E543, E543 food additive
-nl:E543, Natriumcalciumpolyfosfaat
+nl:E543, E543 food additive
```

Comment:
is that a desired change ?

Reply:
yes. see @aleene 's comment:

#comment:en:The same as E452iii, put any synonyms there
en:E543, E543 food additive